core/src/test/java/org/infinispan/api/mvcc/PutForExternalReadTest.java

```diff
@@ -175,7 +168,7 @@ public void testExceptionSuppression() throws Exception {
       Cache<String, String> cache1 = cache(0, CACHE_NAME);
       Cache<String, String> cache2 = cache(1, CACHE_NAME);
 
-      cache1.getAdvancedCache().addInterceptorBefore(new CommandInterceptor() {
+      assert cache1.getAdvancedCache().addInterceptorBefore(new CommandInterceptor() {
```

Comment:
The `addInterceptor` method is not conditional, it either adds the interceptor or throws an exception.